### PR TITLE
OWNERS: Add OWNERS file

### PR DESCRIPTION
**Pull request by sameo**
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ comment line:
  - +1
  - Approve
 
+### Project maintainers
+
+The Clear Containers runtime maintainers will be the ones accepting or rejecting any pull request. They are listed in the OWNERS files, and there can be one OWNERS file per directory.
+
+The OWNERS files split maintainership into 2 categories: reviewers and approvers. All approvers also belong to the reviewers list and there must be one approval from one member of each list for a pull request to be merged.
```
Comment (grahamwhaley) [inline CONTRIBUTING.md]: suggestion: 'must be at least one approval from a member of each list'

**Comment by grahamwhaley** [comment]
minor comment.
lgtm

**Comment by devimc** [comment]
lgtm

**Pull request merged**